lara-typescript/webpack.config.js

```diff
@@ -1,15 +1,18 @@
 'use strict';
 const ForkTsCheckerWebpackPlugin = require('fork-ts-checker-webpack-plugin');
+const CopyPlugin = require('copy-webpack-plugin');
 
 module.exports = (env, argv) => {
   return {
     context: __dirname, // to automatically find tsconfig.json
     devtool: 'source-map',
-    entry: './src/lara-plugin-api.ts',
+    entry: './src/index.ts',
     mode: 'development',
     output: {
-      filename: 'lara-plugin-api.js',
-      library: 'LARA_V3',
+      filename: 'lara-typescript.js',
+      // Temporarily disabled, as it conflicts with V2 definition. Instead, src/index.ts file exports library
+      // manually by extending LARA namespace. When lara-plugin-api-V2 is removed, this can be uncommented.
+      // library: 'LARA',
```

Comment:
If we go with LARA.PluginAPI_V3 is this comment still relevant? If the line is un-commented will it mess up plugins that are expecting the api at LARA.PluginAPI_V3?  Sorry I haven't taken the time to figure out this output option in wepack.

Reply:
It is still relevant, unfortunately. The problem is that the old API is available under `LARA` namespace. And this new code still uses top-level `LARA` namespace. Webpack seems to create a new object instead of trying to check if it already exists and try to expand it. So, V2 namespace will be overwritten by this one.

In other words, we would need to change top-level namespace, not just the `PluginAPI` part. But that's a tiny thing, once we do V2 cleanup, this should be 30 seconds of work.

Reply:
OK, I think I understand now. So once this lara-typescript package contains all of the javascript that makes up the `LARA` object and there is no javascript in LARA outside of this package, then we can make the switch.  And this switch will not affect any plugins because the api will still be at LARA.PluginAPI_V3.

Reply:
Exactly.